doc-surrealdb_versioned_docs/version-2.x/surrealql/statements/define/access/record.mdx

```diff
@@ -81,16 +81,12 @@ DEFINE TABLE user SCHEMAFULL
   -- The record of the user being queried must match the one identified in the token
   -- Only matching records will be changed or returned
   AND id = $auth.id
+  -- Allow privileged tokens to query any user
+  OR $token.privileged = true
 ;
-
-DEFINE INDEX email ON user FIELDS email UNIQUE;
-DEFINE FIELD email ON user TYPE string ASSERT string::is::email($value);
-DEFINE FIELD name ON user TYPE string;
-DEFINE FIELD nickname ON user TYPE string;
-DEFINE FIELD picture ON user TYPE string;
 ```
 
-You may also use permissions clauses to perform additional verification on other JWT claims (e.g. verifying that the iss claim matches a specific principal using $token.iss) that may be required or recommended by a the provider of the token.
+You may also use permissions clauses to perform additional verification on other JWT claims that may be required or recommended by the provider of the token, such as verifying that the `iss` claim matches a specific principal using `$token.iss`. However, this kind of logic may be better suited for the [`AUTHENTICATE`](/docs/surrealdb/2.x/surrealql/statements/define/access/record#with-authenticate-clause) clause, which is only executed when the token is validated before an authenticated session is established instead of in every query and for each record.
```

Comment:
```suggestion
You may also use permissions clauses to perform additional verification on other JWT claims that may be required or recommended by the provider of the token, such as verifying that the `iss` claim matches a specific principal using `$token.iss`. However, this kind of logic may be better suited for the [`AUTHENTICATE`](#with-authenticate-clause) clause, which is only executed when the token is validated before an authenticated session is established instead of in every query and for each record.
```

Reply:
Same page so the anchor tag is alright